docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,272 @@
+};
+type SetDissolveTimestamp = record { dissolve_timestamp_seconds : nat64 };
+```
+
+To move a neuron from non-dissovling into dissovling state, and start the timer, use `StartDissolving`. No additional argument are required.
+
+To move a neuron from dissolving into non-dissolving state, and stop the timer at the dissolve state that the neuron has, use `StopDissolving`.
+
+No matter in which dissolve state a neuron is, its dissolve delay can be increased up to a maximum of eight years.
+To do so, there are two options.
+You can use `IncreaseDissolveDelay` and specify by the argument `additional_dissolve_delay_seconds` by how many seconds the current remaining dissolve delay should be increased. For example, if a neuron has 6 months dissolve delay, choosing the argument 2 months (in seconds) for `IncreaseDissolveDelay`, would result in a neuron with 8 months dissolve delay. 
+Alternatively, you can specify how much dissolve delay a neuron should have after the operation by using `SetDissolveTimestamp` and setting a target time stamp `dissolve_timestamp_seconds` in seconds from the Unix epoch.
+To achieve the same as above, compute the timestamp of now plus 8 months and provide this as the input `dissolve_timestamp_seconds`. 
+In both cases, the dissolve delay can only be increased but never decreased.
+
+_Required permissions:_
+Only a neuron's controller can start or stop dissolving a neuron or increase its dissolve delay.
+
+
+### Spwaning a neuron's rewards
+When a neuron's maturity has risen above a threshold, one can spawn a portion of the maturity. In a first step, this creates a new neuron in a special spawning state containing the spawned maturity. After 7 days, the maturity is replaced with newly minted ICP, taking some [maturity modulation function](https://wiki.internetcomputer.org/wiki/Maturity_modulation#:~:text=The%20maturity%20modulation%20function%20introduces,NNS%20neurons%20and%20SNS%20neurons.) into account. In the end, the new neuron will be dissolved and have these ICP as stake - which can then be disbursed to any account.
+The new neuron can remain controlled by the same principal as the parent neuron, or be assigned to a new principal.
+
+The following command spawns a neuron.
+```
+type Spawn = record {
+  percentage_to_spawn : opt nat32;
+  new_controller : opt principal;
+  nonce : opt nat64;
+};
+```
+Specify by `perpercentage_to_spawn` the portion of the maturity that should be spawn into a neuron in percentage. This should be a value between 1 and 100 (inclusive). 
+If the spawned neurons should have a different controller than the parent neuron (from which you spawn the maturity), you can optionally define a different controlling principal by `new_controller`. Otherwise, the spawned neuron will have the same controller as the parent neuron. 
+Similarly to claiming a neuron (see above), you need to specify the new neuron's ledger address and choose a nonce for this in the last argument `nonce`.
```

Comment:
What happens if the nonce is not specified? What if it specifies an already existing neuron?

Reply:
In that case it seems that a random nonce is generated : https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/nns/governance/src/governance.rs?L2906

Good point - I will add this!